mne/decoding/tests/test_receptive_field.py

```diff
@@ -303,7 +310,8 @@ def test_time_delaying_fast_calc():
                          [31, 19, 7, 112, 74, 35],
                          [15, 10, 5, 55, 35, 25]])
     assert_allclose(Xt_X, expected)
-    x_xt = _compute_corrs(X, np.zeros((X.shape[0], 1)), smin, smax + 1)[0]
+    x_xt = _compute_corrs(X, np.zeros((X.shape[0], 1)), smin, smax + 1,
+                          n_jobs)[0]
```

Comment:
These tests look highly redundant ... would it not make sense to define a function earlier, save lines and make stuff more readable?